<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-AJJC2RF" ignored="0" name="Test A" tests="2" failures="2" timestamp="2020-11-10T23:41:45 CST" time="6.868" errors="0">
  <testcase name="testA" time="3.033" classname="suitea.TestA">
    <failure type="java.lang.AssertionError" message="xyz error">
      <![CDATA[java.lang.AssertionError: xyz error
at suitea.TestA.testA(TestA.java:18)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testA -->
  <testcase name="testA" time="3.018" classname="suitea.TestA">
    <failure type="java.lang.AssertionError" message="xyz error">
      <![CDATA[java.lang.AssertionError: xyz error
at suitea.TestA.testA(TestA.java:18)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testA -->
</testsuite> <!-- Test A -->
